### DISCUSSION TITLE
Mongo's ObjectId in appServices

### DISCUSSION THREAD

**Discussion by MaheshkumarSundaram** [bpmnServer/bpmn-server#230] [Q&A]
Hi @ralphhanna,

In `appServices`, when a service is called, I can access the data of the whole instance from `context`. But within `context`, I couldn't find the `ObjectId` of the workflow instance!

Is there any way to get that `ObjectId` within the service?

**Comment by ralphhanna** [comment]
Hi @MaheshkumarSundaram ,

will this work:
```ts
context.instance.id
```
![image](https://github.com/user-attachments/assets/617b51a3-a9ce-494b-b8d8-6f6bd1aa53dd)

**Reply by MaheshkumarSundaram** [reply]
@ralphhanna 

Doesn't `context.instance.id` refer to the instance id?

I am referring to the `ObjectID` (`_id`) generated by Mongo for the workflow instance.

**Comment by ralphhanna** [comment]
MongoDB ObjectID is not used.
